posts/2019-07-04-Rust-1.36.0.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "Announcing Rust 1.36.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.36.0. Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.36.0 is as easy as:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate page on our website, and check out the [detailed release notes for 1.36.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1360-2019-07-04
+
+## What's in 1.36.0 stable
+
+This release brings many changes, including the stabilization of the [`Future`] trait, the [`alloc`][alloc-crate] crate, the [`MaybeUninit<T>`] type, [NLL on 2015][felix-blog], a new `HashMap<K, V>` implementation, and [`--offline`] support in Cargo. Read on for a few highlights, or see the [detailed release notes][notes] for additional information.
+
+### The [`Future`] is here!
+
+[`Future`]: https://doc.rust-lang.org/std/future/trait.Future.html
+[pr-future]: https://github.com/rust-lang/rust/pull/59739
+
+In Rust 1.36.0 the long awaited [`Future`] trait has been [stabilized][pr-future]!
+
+With this stabilization, we hope to give important crates, libraries, and  the ecosystem time to prepare for `async` / `.await`, which we'll tell you more about in the future.
```

Comment:
It isn't; we hope to release it 1.38.0 but I think we shouldn't say so in a blog post unless we are 100% sure it will happen to not raise false hopes.